app/views/external/contact.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Contact Page</h1>
-<p>Contact page stuff goes here</p>
\ No newline at end of file
+<div class="content" style="display: flex;">
+    <div class="container" style="margin: auto; display: flex; flex-direction: column">
+        <h1 style="margin: auto;">Contacts</h1>
+        <img src="/assets/default.jpg" alt="profile" style="width: 30vw; margin: auto; max-width: 250px;">
+        <p style="margin: auto;">Below you can find our email and social links, be sure to follow MCHSO on Facebook!</p>
+        <ul style="margin: auto">
+            <li><a href="mailto:hannahbartosh@tamu.edu">Email</a></li>
+            <li><a href="https://www.facebook.com/MCHSOaggies/">facebook</a></li>
```

Comment:
Let's use `link_to` here instead of `<a>` so we can be more consistent with styling later on.